velox/expression/tests/SparkExpressionFuzzerTest.cpp

```diff
@@ -51,6 +51,9 @@ int main(int argc, char** argv) {
       "chr",
       "replace",
       "might_contain",
+      // This function relies on an externally configured setting:
+      // spark.partition_id, which cannot currently be established.
```

Comment:
What would it take to extend the Fuzzer to allow specifying configuration properties?

Reply:
We need pass the config with `spark.partition_id` from this class to `ExpressionVerifier`. Please let me know I should add it in this PR, thanks.

Reply:
It would be nice to extend the Fuzzer to avoid skipping this function. Thanks.

CC: @kgpai @kagamiori

Reply:
Updated, thanks.